## Welcome!

This is a collection of notebooks describing the basic analysis workflow for a 16S rRNA gene amplicon sequencing project - from raw reads to statistics and plots. It is intended for beginners, and therefore includes an introductory section on the R programming language and statistics.

If you find errors/typos or you would like a notebook/section on a specific topic, make a [github issue](https://github.com/Russel88/amplicon_data_analysis/issues) or throw me an email: russel (at) biostat (dot) xyz

The notebooks uses an example dataset, which is a simulated dataset, which is supposed to look like an infant gut microbiome dataset. It is available for download [here](https://github.com/Russel88/amplicon_data_analysis/raw/master/data/physeq.RData).

Only for KU students:
* [Intro to ERDA](html/ERDA.html)

Background material:
* [Intro to R and RStudio](html/Rstudio.html)
* [Intro to R programming language](html/R.html)
* [Statistics 101](html/stats.html)
* [Statistics 102](html/stats2.html)
* [The phyloseq object](html/phyloseq_object.html)
* [Phyloseq operations](html/phyloseq_operations.html)
* [Compositionality and rarefaction](html/compositionality.html)
* [Extra plotting with ggplot2](html/ggplot2.html)

Bioinformatic workflow:
* [Preparation of reads and databases](html/prepare.html)
* [DADA2 - ASV inference](html/dada2.html)

Analysis workflow:
* [Alpha diversity](html/alpha.html)
* [Beta diversity](html/beta.html)
* [Differential abundance - Basics](html/da.html)

### Extra analysis
This section contains notebooks for more specific and/or advanced data presentations or analyses. 

##### Analysis
* [PERMANOVA effect sizes](html/omegasq.html)
* [Differential abundance - Extended](html/da2.html)
* [Microbial association networks](html/network.html)

##### Plotting
* [Venn diagram of shared taxa](html/venn.html)
* [Pretty heatmaps](html/pheatmap.html)

### Info
* [Install packages](html/packages.html)
* [Software versions](html/versions.html)


